tools/dataset_tools/config_utils.py

```diff
@@ -0,0 +1,265 @@
+#!/usr/bin/env python
+
+import json
+import os
```

Comment:
You are doing a lot of path based manipulation here. Consider looking at Pathlib.